tests/test_message.py

```diff
@@ -12,6 +13,8 @@ def test_render(mocked_settings):
     specified context.
     '''
     message = Message(123, 'Telegram', 'Bottery', 'text', 1506191745, 'raw')
+    assert message.datetime == datetime.datetime(2017, 9, 23, 18, 35, 45)
+
```

Comment:
Shouldn't we have a specific test for the `Message.datetime`?

Reply:
done